### DISCUSSION TITLE
ScriptReference/PlayerSettings.iOS-appleDeveloperTeamID

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#208] [Notes]
# ScriptReference/PlayerSettings.iOS-appleDeveloperTeamID

https://docs.unity3d.com/ScriptReference/PlayerSettings.iOS-appleDeveloperTeamID.html

**Comment by mvi** [comment]
For personal licenses you can find your Team ID by:  
- Open Xcode  
- Setting your team to personal, then save  
- Open Unity-iPhone.xcodeproj/project.pbxproj in a text editor  
- Find the line that starts with 'DevelopmentTeam = ' and ends in a hash, this hash is your Team ID  
Note that the Team ID setting is shared between all projects on a machine, it is not stored in each project's ProjectSettings.asset file like most of the PlayerSettings.iOS class.
